modular_skyrat/modules/medical_designs/medical_designs.dm

```diff
@@ -8,3 +8,13 @@
 		RND_CATEGORY_EQUIPMENT + RND_SUBCATEGORY_EQUIPMENT_MEDICAL
 	)
 	departmental_flags = DEPARTMENT_BITFLAG_MEDICAL | DEPARTMENT_BITFLAG_SCIENCE
+
+/datum/design/surgery/healing/robotic_healing_upgrade
+	name = "Repair robotic limbs Upgrade"
+	surgery = /datum/surgery/robot_healing/upgraded
+	id = "robotic_heal_surgery_upgrade"
+
+/datum/design/surgery/healing/robotic_healing_upgrade_2
+	name = "Repair robotic limbs Upgrade"
+	surgery = /datum/surgery/robot_healing/experimental
```

Comment:
They have the same name, is that intended?

Reply:
sorta, but a different name isnt a bad idea